includes/html/common/worldmap.inc.php

```diff
@@ -130,6 +130,123 @@
 marker.bindPopup(title);
     markers.addLayer(marker);\n";
     }
+
+    if (Config::get('network_map_show_on_worldmap', true)) {
```

Comment:
, true   is not needed because default value is defined via config_definitions file